### DISCUSSION TITLE
希望18.06及18.06-5.4两Branch添加P2W R619AC-128M支持

### DISCUSSION THREAD

**Discussion by BestHH** [immortalwrt/immortalwrt#702] [Q&A] [answered]
目前只有21.02branch添加了P2W R619AC-128M支持，18.06两个branch只有P2W R619AC，暂时还不会pull request，希望添加支持，谢谢！

**Comment by 1715173329** [comment, marked as answer]
18.06 默认就是 128M，无 64M 支持。

**Reply by BestHH** [reply]
自行编译的immortal18.06，luci sysupgrade无法升级，提示固件不支持，immortal21.02的P2W R619AC-128M就可以直接升级，都是luci界面升级，非U-Boot，本机固件为lean源码编译的P2W R619AC-128M

**Reply by 1715173329** [reply]
supported_devices 不同，命令行输入 sysupgrade -F -n <filename> 刷入，或 u-boot 刷入。

**Reply by BestHH** [reply]
好的，感谢回复！
